02-Homework/script.js

```diff
@@ -0,0 +1,56 @@
+// Assignment Code
+var generateBtn = document.querySelector('#generate');
+
+function generatePassword() {
+  // PLEASE ADD ALL YOUR CODE HERE
+}
+alert ("text");
+var text
+var intiger
+if ( text = intiger) {
+"check is within range"
+} else {
+  alert and break
+}
+
+var confirm; (lowercase)
+var confirm; (uppercase)
+var confirm; (number)
+var confirm; (special)
+
+if (if all is false) {
+  alert and break
+
+} else {
+  return criteria
+}
+
+const (uppercase) "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+const (lowercase) "abcdefghijklmnopqrstuvwxyz"
+const (number) "0123456789"
+const (special) " !#$%&'()\*+,-./:;<=>?@[\]^\_`{|}~ "
+
+function getrandomcharacter () {
+
+  Array; "uppercase"
```

Comment:
Again, we should never see the `;` followed by something on the same line, as `;` denotes the end of a line.